src/components/Alert/__tests__/__snapshots__/Alert.spec.js.snap

```diff
@@ -2,13 +2,13 @@
 
 exports[`<Alert> that renders an alert should add classes when props are changed 1`] = `
 <div
-  className="Alert Alert--context_bad"
+  className="Alert Alert--context Alert--context_bad"
 >
   <div
-    className="Alert__content Alert__content--context_bad"
+    className="Alert__content Alert__content--context Alert__content--context_bad"
   >
     <strong
-      className="Alert__title Alert__title--context_bad"
+      className="Alert__title Alert__title--context Alert__title--context_bad"
```

Comment:
I noticed some classnames that are not used at all, they don't exist. Ex: `Alert__title--context` , `Alert__content--context`. Is it the desired behavior?

Reply:
I also noticed it. That's because this `identity-obj-proxy`thing that we use in jset to mock css imports.

I created a ticket for it here: https://trello.com/c/1J8lsZsE/78-make-jest-use-real-style-declarations-instead-of-mocked-one